data/microos/butane/script

```diff
@@ -79,28 +82,63 @@ test_prepare() {
     echo "combustion: test_prepare function ran OK" > /dev/kmsg
 }
 
+can_encrypt() {
+	[ -c "/dev/tpm0" ] && command -v disk-encryption-tool >/dev/null
+}
+
 if [ "${1-}" = "--prepare" ]; then
     test_prepare
+
+	if can_encrypt; then
```

Comment:
Let's fix the indentation here

Reply:
sorry, i have missed your comment. Funny that it looked fine in vim. :shrug: